cookbooks/proxyfs/files/default/usr/bin/unmount_and_stop_pfs

```diff
@@ -69,7 +71,9 @@ else
     # sudo /usr/sbin/service winbindd stop
     sudo /usr/sbin/service proxyfsd stop
     await_proxyfsd_shutdown
-    sudo /usr/bin/swift-init main stop
-    # sudo /usr/sbin/service swift stop
+
+    # NOTE: We purposely stop Swift with "swift-init all stop" and not
+    # "swift-init main stop".   See start_and_stop_pfs for more details.
```

Comment:
Not very important, but I guess you mean `start_and_mount_pfs` instead of `start_and_stop_pfs`.